framework/web/Request.php

```diff
@@ -1444,8 +1419,8 @@ public function validateCsrfToken($token = null)
     /**
      * Validates CSRF token
      *
-     * @param string $token
-     * @param string $trueToken
+     * @param string $token The masked client-supplied token.
```

Comment:
would be logical to name the parameter corresponding to its description $clientSuppliedToken